CONTRIBUTING.md

```diff
@@ -1,14 +1,171 @@
-# Contributing
+# Contributing to OpenTelemetry Protocol (OTLP) Specification
 
-Read OpenTelemetry project [contributing
-guide](https://github.com/open-telemetry/community/blob/main/guides/contributor/README.md)
-for general information about the project.
+## Introduction
+
+Welcome to the OpenTelemetry Protocol (OTLP) Specification repository! 🎉
+
+Thank you for your interest in contributing to this project. This repository contains the OTLP specification and the corresponding Language Independent Interface Types (`.proto` files). Your contributions help improve the interoperability of OpenTelemetry clients across various programming languages.
+
+If you have any questions, feel free to ask in the community channels. We're happy to help! 😊
 
 ## Prerequisites
 
-- `Docker`
+Before getting started, ensure you have the following installed:
+
+* **Docker** – [Install Docker](https://docs.docker.com/engine/install/)
+* **Make** (for running development tasks)
+* **Protobuf Compiler (protoc)** – [Install protoc](https://grpc.io/docs/protoc-installation/)
+
+Additional Notes:
+
+* Ensure your environment supports the required dependencies for building and testing .proto files.
+* Windows users may need [Git Bash](https://gitforwindows.org/) for better compatibility.
+
+## Workflow
+
+We follow a structured workflow to ensure smooth collaboration:
+
+### Commit Message Format
+
+* Use descriptive commit messages (e.g., `fix(proto): resolve issue with trace spec`)
+* Follow [Conventional Commits](https://www.conventionalcommits.org/) where possible.
+
+### Pull Request Guidelines
+
+- Fork the repository and create a new branch.
+- Follow the coding guidelines before submitting your PR.
+- Ensure tests pass locally before pushing.
+- Link relevant issues in the PR description.
+
+---
+
+## Local Setup & Development
+
+To set up your local development environment:
+
+```bash
+# Clone the repository
+git clone https://github.com/open-telemetry/opentelemetry-proto.git
+cd opentelemetry-proto
+
+# Generate all implementations
+make gen-all
+```
+
+To validate your changes:
+
+```bash
+make check
+```
+
+To update dependencies:
+
+```bash
+make update
```

Comment:
Dito, there is no `update` target. Are instructions copy/pasted from somewhere else? Please remove irrelevant parts.